ufc/bandit-flags-v1.json

```diff
@@ -218,6 +243,14 @@
                 "variationValue": "banner_bandit"
             }
         ],
+        "car_bandit": [
```

Comment:
missing `car_bandit` in `banditReferences`. Hopefully we can fully deprecate and remove `bandits` at some point in the future... maybe when everyone is on UFCv2 (whatever that may look like)

Reply:
GREAT catch! I'm embarassed I forgot that 🤦 

Added to `car_bandit` to `banditReferences` and also fixed up the incorrect allocation keys in there as well.